lib/bank/account.rb

```diff
@@ -0,0 +1,37 @@
+module Bank
+  class Account
+    # attr_accessor creates an implicit balance method
+    attr_accessor :id, :balance
+
+    def initialize(id, initial_balance)
+      raise ArgumentError, "Negative balance is invalid." if initial_balance.to_i < 0
+
+      @id = id
+      @balance = initial_balance.to_i
```

Comment:
I thought about that… since most of these methods are responding to ATM transactions, I'm assuming you would only deposit/withdraw whole dollars. I'll double check that my check transaction methods allow for this though.